app/models/user.rb

```diff
@@ -5,14 +5,15 @@ class User < ActiveRecord::Base
 
   PERMITTED_ATTRS = [:bio, :email, :handle, :hide_email, :location, :password, :website].freeze
 
+  before_destroy :yank_gems
```

Comment:
Wait, why are we yanking gems? I would assume that removing my user does not remove any gems.